lib/search.rb

```diff
@@ -989,7 +986,7 @@ def posts_query(limit, type_filter: nil, aggregate_search: false)
 
       if aggregate_search
         posts = posts.select("#{data_ranking} rank", "topics.bumped_at topic_bumped_at")
-          .order("rank DESC", "topic_bumped_at DESC")
+          .order("rank DESC", "topic_bumped_at DESC", "posts.id DESC")
```

Comment:
When two posts in a topic has the same rank, we should order by the latest post.